politeiawww/user/mysql/mysql.go

```diff
@@ -1011,9 +1019,137 @@ func (m *mysql) Close() error {
 	return m.userDB.Close()
 }
 
+// EmailHistoriesSave creates or updates the email histories.
+//
+// EmailHistoriesSave satisfies the user MailerDB interface.
+func (m *mysql) EmailHistoriesSave(histories map[uuid.UUID]user.EmailHistory) error {
```

Comment:
This get, update, save should be done using a transaction.

https://golang.org/pkg/database/sql/#DB.BeginTx